app/assets/javascripts/cart.js

```diff
@@ -137,20 +126,53 @@ $(document).ready(function() {
         updateCart();
     });
 
+    $(".cart-span").on("click", ".del", function() {
+        var items = JSON.parse(localStorage["getmyshop_shoppping_cart_items"]);
+        var element_id = $(this).attr("id");
+        var cost = 0;
+        if (items.length == 1) {
+            items[0] = undefined;
+            location.reload();
+            Materialize.toast("Your cart has been emptied!", 3000);
+        } else {
+            for (var i in items) {
```

Comment:
The body of a for in should be wrapped in an if statement to filter unwanted properties from the prototype.